lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -103,7 +103,6 @@ describe('ReportUIFeatures', () => {
     global.I18n = undefined;
     global.ReportUIFeatures = undefined;
     global.matchMedia = undefined;
-    global.self.matchMedia = undefined;
```

Comment:
I.... have no idea how this was working ... but I am extremely curious.

maybe before `global.self === global` and now it does not? In Chrome `self === globalThis.self`, so circular global object references makes some sort of sense, but maybe Jest decided to remove that behavior for `global`.